modules/oas-page-builder/src/services/pageBuilder.ts

```diff
@@ -148,7 +152,40 @@ async function getOASpec({
 
     const path = `${output}/${pageSlug}${filePathExtension}/index.html`;
     const finalFilename = normalizePath(path);
-    await redocExecutor.execute(spec, finalFilename, buildOptions);
+
+    let versionOptions: RedocVersionOptions | undefined;
+
+    if (resourceVersions && apiVersion) {
+      const rootUrl = `${siteUrl}/${pageSlug}`;
```

Comment:
I think keeping this forward slash is ok just in case, but we should run this through the util `normalizeUrl` because right now it has two forward slashes in a row